src/components/molecules/Card-Home/Card.scss

```diff
@@ -0,0 +1,32 @@
+@import "../../../assets/styles/main.scss";
+.main__card{
+        &_subcontainer{
+            max-width: $width-content-container;
+            box-shadow: $home-card-container-shadow;
+            display: flex;
+            justify-content: space-between;
+            margin: 0 auto;
+            padding: 10px;
```

Comment:
give it in rem and define tokens where posiible

Reply:
Done  Salil